### PR TITLE
Document openURL, tokenStorage and shareSessionWithSystemBrowser

### PR DESCRIPTION
**Pull request by louischan-oursky**
https://github.com/authgear/authgear-server/issues/1503

### CODE REVIEW

File: integrate/force-authentication-on-app-launch.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+You can alter this behavior by switching to a transient storage.
+
+{% tabs %}
+{% tab title="React Native" %}
+```typescript
+import authgear, { TransientTokenStorage } from "@authgear/react-native";
+
+authgear.configure({
+    tokenStorage: new TransientTokenStorage(),
```
Comment (carmenlau) [inline integrate/force-authentication-on-app-launch.md:15]: Should also have `clientID` and `endpoint` in the JS version?

File: integrate/single-sign-on-on-mobile-device.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+1. The end-user installed your app. They signed in in the app. Later on when they visit your website with the system browser, they are signed in.
+2. The end-user have been using your website. One day they decided to install your app. When they are signing in in the app, they will see a continue screen so that they can continue with just a click, without authenticating themselves again.
+
+You can turn on this feature when you configure the SDK.
+
+{% tabs %}
+{% tab title="React Native" %}
+```typescript
+authgear.configure({
+    shareSessionWithSystemBrowser: true,
```
Comment (carmenlau) [inline integrate/single-sign-on-on-mobile-device.md:14]: Same here

**Review by carmenlau — COMMENTED** [review]
Overall looks good to me!

**Review by fungc-io — APPROVED** [review]
Overall looks good :+1:
I will change some wordings after merge.

**Pull request merged**